stylix/target.nix

```diff
@@ -34,28 +34,83 @@
   config.lib.stylix =
     let
       cfg = config.stylix;
+      self = config.lib.stylix;
```

Comment:
I think it would be clearer without this `self` alias? But I am also fine with keeping it.

Reply:
`self` is a common pattern, and it ends up less verbose than explicitly using `config.lib.stylix` repeatedly. But I don't mind changing it if you have a strong opinion. If you don't feel strongly I'll leave it as-is.